pkg/sharing/secret_exports.go

```diff
@@ -90,12 +90,12 @@ type SecretMatcher struct {
 // MatchedSecretsForImport filters secrets export cache by the given criteria.
 // Returned order (last in the array is most specific):
 //   - secret with highest weight? (default weight=0), or
-//     - secret within the same namespace
-//       - secret with specific namespace
-//       - secret with wildcard namespace match
-//     - secret within other namespaces
-//       - secret with specific namespace
-//       - secret with wildcard namespace match
```

Comment:
I guess my IDE didn't like the spacing? I can revert it if it's important... seems like not so much though